cross-platform/react-app/src/frontend/Screens/HomeScreen.tsx

```diff
@@ -26,10 +28,19 @@ export function HomeScreen(props: HomeScreenProps) {
   const homeLabel = React.useMemo(() => i18n("HomeScreen", "Home"), []);
   const snapshotIModelsLabel = React.useMemo(() => i18n("HomeScreen", "SnapshotIModels"), []);
   const hubIModelsLabel = React.useMemo(() => i18n("HomeScreen", "HubIModels"), []);
+  const haveBackButton = React.useMemo(() => MobileCore.getUrlSearchParam("haveBackButton") === "YES", []);
```

Comment:
It would be better if this was added to HomeScreenProps.

Reply:
Done.